demo/src/app/examples/advanced/tabs-bootstrap/app.component.ts

```diff
@@ -0,0 +1,92 @@
+import { Component } from '@angular/core';
+import { FormlyFormOptions, FormlyFieldConfig } from '@ngx-formly/core';
+import { FormGroup } from '@angular/forms';
+
+export interface StepType {
+  label: string;
+  fields: FormlyFieldConfig[];
+}
+
+@Component({
+  selector: 'formly-app-example',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.css'],
+})
+export class AppComponent {
+
+  form = new FormGroup({});
+  model = {};
+  options: FormlyFormOptions = {};
+
+  fields: FormlyFieldConfig[] = [
+    {
+      type: 'tabBootstrap',
+      fieldGroup: [],
+      templateOptions: {
+        steps: [
+          {
+            label: 'Personal data',
+            fields: [
+              {
+                key: 'firstname',
+                type: 'input',
+                templateOptions: {
+                  label: 'First name',
+                  required: true,
+                },
+              },
+              {
+                key: 'age',
+                type: 'input',
+                templateOptions: {
+                  type: 'number',
+                  label: 'Age',
+                  required: true,
+                },
+              },
+            ],
+          },
+          {
+            label: 'Destination',
+            fields: [
+              {
+                key: 'country',
+                type: 'input',
+                templateOptions: {
+                  label: 'Country',
+                  required: true,
+                },
+              },
+            ],
+          },
+          {
+            label: 'Day of the trip',
+            fields: [
+              {
+                key: 'day',
+                type: 'input',
+                templateOptions: {
+                  type: 'date',
+                  label: 'Day of the trip',
+                  required: true,
+                },
+              },
+            ],
+          },
+        ],
+      },
+    },
+  ];
+
+constructor() {
+
+  // I would like to do this login in the template.
+  this.fields[0].fieldGroup = ([]).concat(this.fields[0].templateOptions.steps.map((val) => {
```

Comment:
this is useless, declare those step directly inside `fieldGroup`

Reply:
Where I put the labels of tabs in the fieldGroup?
The "step" array had the structure label fields.

Reply:
I had in the 
```
 templateOptions: {
        labels: ['Personal data', 'Destination', 'Day of the trip'],
      },
```

Reply:
```patch
- <span class="h6">{{ getStepTitle(index) }}</span>
+ <span class="h6">{{ f.templateOptions.label }}</span>
```

Reply:
I get your point. Ok